spreadsheets/spreadsheets-intermediate/importing-data/excel-file-types.md

```diff
@@ -65,3 +65,18 @@ The only difference between `.xltx` and `.xltm` file types is that the one of th
 
 - .xltm
 - .xltx
+
+
+---
+
+## Footnotes
+
+[1:Macros]
+
+Macros are miny programs that do not require any code to make. Their goal is to automate repeatable tasks. You go to the Spreadsheet menu, select "Record Macro", and it will start recording your actions. From there, you can create different macros. For example, you can add a custom shortcut and name to each macro. Also, you can schedule macros to execute at specific times. 
```

Comment:
hmm, maybe split these into two paragraphs?

```suggestion
Macros are mini-programs that do not require any code to make. Their goal is to automate repeatable tasks.

To create a macro, go to the Spreadsheet menu, select "Record Macro", and it will start recording your actions. From there, you can create different macros. You can even add a custom shortcut and name to each macro. Also, you can schedule macros to execute at specific times. 
```